src/main/scripts/bin.js

```diff
@@ -32,3 +34,27 @@ console.log(
 )
 
 console.log(`cd ${repoName} && npm run dev`)
+
+console.log(`Author: ${authorName}`)
+
+// Dynamically update package.json
+const packageJsonPath = `${repoName}/package.json`
+
+try {
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'))
+
+  // Update package.json fields
+  packageJson.name = repoName
+  packageJson.author = authorName
+  packageJson.version = '0.0.1'
+
+  // Write the updated package.json back to the file
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2))
+
+  console.log('Updated package.json with dynamic values.')
```

Comment:
it is not necessary, because it is not an instruction

Reply:
this line is missing! After remove it, run "npm run format:fix" to run prettier here

Reply:
@rubemfsv you are talking about whicj line ?